File: Backend/UAM-LIFT/src/main/java/ni/edu/uam/UAM_LIFT/models/Destino.java
package ni.edu.uam.UAM_LIFT.models;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.hibernate.annotations.SQLDelete;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Table(name = "destinos")
@SQLDelete(sql = "UPDATE destinos SET estado = false WHERE id = ?")
public class Destino {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(nullable = false, length = 100)
    private String nombre;
    @Column(nullable = false)
    private boolean universidad = true;
    @Column(nullable = false)
    private boolean estado = true;
}

File: Uam Lift/app/src/main/java/ni/edu/uam/uamlift/models/Destino.kt
package ni.edu.uam.uamlift.models

import com.google.gson.annotations.SerializedName

data class Destino(
    @SerializedName("id")
    val id: Long? = null,

    @SerializedName("nombre")
    val nombre: String,

    @SerializedName("universidad")
    val universidad: Boolean = false,

    @SerializedName("estado")
    val estado: Boolean = true
)
